codice/ddf
# DDF-2077 Replace Restart Now button with Shutdown Now button

#### What does this PR do?
Replaces the "Restart Now" button on the installer that appears after a user changes the hostname and provides new certificates with a "Shutdown Now" button.
#### Who is reviewing it?
@coyotesqrl 
@ryeats 
@tbatie 
#### Choose 2 committers to review/merge the PR.
@pklinef
@stustison
#### How should this be tested?
Run through the installer changing the hostname, provide the appropriate keystore and truststore (or run with `?dev=true`), and verify the Shutdown Now button.
#### Any background context you want to provide?
N/A
#### What are the relevant tickets?
DDF-2077
#### Screenshots (if appropriate)
Included as part of changes
#### Checklist:
